lib/shared/cache.js

```diff
@@ -42,16 +62,34 @@ function getCaches(cachedir, models, all) {
     }
 
     for (const file of fs.readdirSync(cachedir)) {
-        const [, slug] = file.match(matchCacheFilename) || [];
+        const fullPath = path.join(cachedir, file);
+        const cacheInfo = genCacheInfo(fullPath);
+        const { slug } = cacheInfo;
         const modelConfig = slug && models.find(model => model.slug === slug);
 
         if (modelConfig) {
-            const fullPath = path.join(cachedir, file);
             const stat = fs.statSync(fullPath);
             const cacheAge = Date.now() - stat.mtime;
 
-            if (all || !modelConfig.cacheTtl || modelConfig.cacheTtl > cacheAge) {
-                cachesByModel[slug].push(genCacheInfo(slug, fullPath, stat));
+            if (all) {
+                cachesByModel[slug].push(cacheInfo);
+            } else if (
+                !modelConfig.cacheTtl ||
```

Comment:
Single line conditions, please

Reply:
Currently, if not `all` and model `cacheAge` less than `modelConfig.cacheTtl` it will not on the list, but must be.
So, conditions in this block doesn't work as intended.